metaserver/mongoDB.go

```diff
@@ -0,0 +1,471 @@
+package metaserver
+
+import (
+	"skybin/core"
+
+	"github.com/globalsign/mgo"
+)
+
+const dbName = "skybin"
```

Comment:
We should probably make this configurable in the future.

Reply:
👍